src/rtextures.c

```diff
@@ -3979,22 +4017,29 @@ Texture2D LoadTexture(const char *fileName)
 
 // Load a texture from image data
 // NOTE: image is not unloaded, it must be done manually
-Texture2D LoadTextureFromImage(Image image)
-{
-    Texture2D texture = { 0 };
+Texture2D LoadTextureFromImage(Image image) {
+  Texture2D texture = {0};
 
-    if ((image.width != 0) && (image.height != 0))
-    {
-        texture.id = rlLoadTexture(image.data, image.width, image.height, image.format, image.mipmaps);
-    }
-    else TRACELOG(LOG_WARNING, "IMAGE: Data is not valid to load texture");
+  if ((image.width != 0) && (image.height != 0)) {
+    texture.id = rlLoadTexture(image.data, image.width, image.height,
+                               image.format, image.mipmaps);
+  } else
+    TRACELOG(LOG_WARNING, "IMAGE: Data is not valid to load texture");
 
-    texture.width = image.width;
-    texture.height = image.height;
-    texture.mipmaps = image.mipmaps;
-    texture.format = image.format;
+  texture.width = image.width;
+  texture.height = image.height;
+  texture.mipmaps = image.mipmaps;
+  texture.format = image.format;
```

Comment:
Hello,
Please revert the changes on `LoadTextureFromImage()` as it breaks current [code conventions](https://github.com/raysan5/raylib/wiki/raylib-coding-conventions).
Best regards.